### ISSUE TITLE
Will resnet50 or pvanet works better?

### ISSUE THREAD

**Issue by baby313** [ZJULearning/pixel_link#29]
I tried resnet50 and pvanet as base nets. However, vgg16 is the best one. I wonder that if I did something wrong in training process? Have you guys tried other base nets?

**Comment by GodOfSmallThings** [comment]
@baby313 
I got worse result too with resnet50. Which endpoints did you use in resnet?

**Comment by aravinthmuthu** [comment]
Pixel_link learns very minimalistic representations with limited receptive fields. Hence ResNet might be an overkill. Has anyone experimented with alexNet as the base model?

**Comment by BowieHsu** [comment]
@aravinthmuthu i try to use modified VGG Net as backbone, after cut a lot of parameters, the final mAP is still at 82.5%, PixelLink network is a little bit tricky.

**Comment by GodOfSmallThings** [comment]
@BowieHsu 
PixelLink is really tricky, never seen this kind of network before. My final mAP is 83.3 with only icdar15 dataset. I tried to raise this score by inserting batch norm or change hyper parameters, but it never succeeded yet so far. I'll share it if there is any improvement.

**Comment by lizzyYL** [comment]
@baby313 @GodOfSmallThings @aravinthmuthu @BowieHsu @comzyh 
Did you use the pretained resnet on Imagenet? and could you share the best score based on resnet?

**Comment by GodOfSmallThings** [comment]
@lizzyYL 
In my case, resnet50 performed worse(~7x%) with paper's training setting. 
And since learning rate is relatively so high(1e-2), I'm not sure ImageNet pretrained model would work.

**Comment by lizzyYL** [comment]
@GodOfSmallThings  Thank you for your reply.
My result is 68%-70%(80000 iterations), and the resnet did not been pretrained.

**Comment by baby313** [comment]
~7x% @ not pretrained  @lizzyYL

**Comment by mn7142** [comment]
@GodOfSmallThings @BowieHsu @lizzyYL 
i can't evaluate my result
please help me to calculate precision and recall on my result

**Comment by lizzyYL** [comment]
@mn7142 You can use the test scripts to build an offline test platform，
or  use the online testing : http://rrc.cvc.uab.es/?ch=4&com=evaluation&task=1&gtv=1

**Comment by ccnankai** [comment]
Replacing vgg with resnet-50, why did it deteriorate? Who can explain  to me?  thanks .

**Comment by mn7142** [comment]
thank you @lizzyYL 
how to evaluate on other dataset?

**Comment by jackshaw** [comment]
Hi，@BowieHsu，can you tell me how does batchnorm influence the performance?

**Comment by northeastsquare** [comment]
Have you add dilation to the second last conv?
